server/data/dataAccess/db.js

```diff
@@ -24,5 +25,25 @@ pool.on('error', error => {
 })
 
 module.exports = {
-  query: (text, params) => pool.query(text, params),
+  pool, // for testing only
+  queryWithoutTransaction: (text, params) => pool.query(text, params),
+  query: (text, params) => {
+    const ns = getNamespace('request.scope')
```

Comment:
could possibly look at extracting out a `requestScope` module to wrap direct access to the namespace